fingerprint2.js

```diff
@@ -105,17 +105,19 @@
       keys = this.hasLiedBrowserKey(keys);
       keys = this.touchSupportKey(keys);
       var that = this;
-      this.fontsKey(keys, function(newKeys){
-        var values = [];
-        that.each(newKeys, function(pair) {
-          var value = pair.value;
-          if (typeof pair.value.join !== "undefined") {
-            value = pair.value.join(";");
-          }
-          values.push(value);
-        });
-        var murmur = that.x64hash128(values.join("~~~"), 31);
-        return done(murmur, newKeys);
+      this.audioKey(keys, function(newKeys){
```

Comment:
Yeap, i did it. No erros found : 
https://www.dropbox.com/s/dmjym44qr0eeaop/Screenshot%202016-05-23%2022.06.22.png?dl=0